vuepress/docs/v6.3.2/tutorials/create/ms/generate-microservices-and-micro-frontends.md

```diff
@@ -38,76 +39,85 @@ npm install -g generator-jhipster-entando@6.3.2
-     - `Would you like to generate micro frontends when creating entities?` (Always)
-     - `Would you like to enable internationalization support` (Up to you)
-     - `Please choose the native language of the application` (Up to you)
-     - `Please choose additional languages to install` (if you picked internationalization)
-     - `Besides JUnit and Jest, which testing frameworks would you like to use?` (Up to you)
-     - `Would you like to install other generators from the JHipster Marketplace?` (No)
-
- 4. Next, add an Entity to your microservice and create the corresponding Micro Frontends. For the tutorial, `Conference` is the name of the entity that will be added to the application.
+3. You'll be presented with a series of prompts to configure your project. These are listed below, with the base values for this tutorial in parentheses. Except where noted in **bold**, you can input what works best for you. Note that hitting "Enter" will select the default option.
+
+    - `What is the base name of your application?` (Up to you)
+
+    - `As you are running in a microservice architecture, on which port would like your server to run? It should be unique to avoid port conflicts.` (8081)
+    - `What is your default Java package name?` (Up to you)
+    - `Which *type* of database would you like to use?` (SQL)
+          - If no database is selected you'll be building a stateless microservice, which is a valid choice, but the rest of this tutorial won't work.
+    - `Which *production* database would you like to use?` (PostgreSQL or MySQL)
+    - `Which *development* database would you like to use?` (H2 with disk-based persistence)
+    - `Do you want to use the Spring cache abstraction?` (Yes, with the Caffeine implementation)
+    - ` Do you want to use Hibernate 2nd level cache?` (Yes)
+    - `Would you like to use Maven or Gradle for building the backend?` (**Maven**) <-- this is required for Entando
+    - `Which other technologies would you like to use?` (**Don't select any other technologies**)
+    - `What name would you give to the bundle to share on an Entando digital-exchange?` (Up to you)
+    - `Which is the organization name to use when publishing the docker image?` (**Enter the name of the organization where you are going to push your Docker image. If you're using your personal Docker hub account enter your username.**) <-- this can be changed later as needed
+    - `Would you like to generate micro frontends when creating entities?` (Always)
+    - `Would you like to enable internationalization support` (Up to you)
+    - `Please choose the native language of the application` (Up to you)
+    - `Please choose additional languages to install` (Up to you)
+    - `Besides JUnit and Jest, which testing frameworks would you like to use?` (Up to you)
+    - `Would you like to install other generators from the JHipster Marketplace?` (No)
```

Comment:
it's the base value. if it's bolded you have to use it and that's made clear; otherwise you can choose whatever you want. "up to you" means there is no base value

Reply:
I wanted things to be bolded cuz it made me nervous